css/styles.css

```diff
@@ -5,3 +5,23 @@ body {
 .car {
   width: 10rem;
 }
+
+.down {
+  transform: rotate(90deg);
+  transition: transform 0.5s;
```

Comment:
You should put `transition` on the `.car` rule so you don;t need to repeat it four times